gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -279,6 +302,8 @@ function GM:PlayerEndVoice(ply, no_reset)
 	end
 
 	if IsValid(ply) and not no_reset then
+		VOICE.UpdatePlayerVoiceVolume(ply)
+
```

Comment:
I don't think we need to update the volume on end, do we?

Reply:
Originally when testing with bots it became useful. I was assuming due to issues where the speaker tags in the top left show the wrong color, that these signals weren't 100% reliable, so having them be set on both edges might prevent any jarring audio volume jumps when someone begins speaking.

Reply:
I think its safe to assume that the Begin function is always called.
So we can remove this, since the volume adjustment is not needed here. 
Also i dont think this should cause any jarring.